lib/cocoapods/installer/xcode/pods_project_generator/pod_target_installer.rb

```diff
@@ -388,6 +393,22 @@ def create_build_phase_to_symlink_header_folders
             eos
           end
 
+          # Creates a build phase to put the static framework archive in the appropriate framework location
+          # Since Xcode does not provide template support for static library frameworks, we've built a static library
+          # of the form lib{LibraryName}.a. We need to move that to the framework location -
+          # {LibraryName}.framework/{LibraryName}.
+          #
+          # @return [void]
+          #
+          def create_build_phase_to_move_static_framework_archive
+            build_phase = native_target.new_shell_script_build_phase('Setup Static Framework Archive')
```

Comment:
@paulb777 somehow I came back to this PR from a Twitter link.

Should we:

- Prefix this script phase with `[CP]` that designates all CocoaPods script phases?
- Do we need to add input/output paths to ensure this is not running over and over again?

Reply:
@dnkoutso Yes and Probably.  I'll try to take a look today, but might not get to it until later next week.

Reply:
@dnkoutso Actually it looks like the pod build phases aren't prefixed, presumably because they're for the Pod target as opposed to modifications to the developer's existing targets. See an example here: https://github.com/CocoaPods/CocoaPods/blob/master/lib/cocoapods/installer/xcode/pods_project_generator/pod_target_installer.rb#L397

Reply:
ah I see. I guess those are fine and dont take that much time. It might be fine for now.